cmd/divergenceSpectrum/divergenceSpectrum.go

```diff
@@ -0,0 +1,128 @@
+// Command Group: "Sequence Evolution & Reconstruction"
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/interval"
+	"github.com/vertgenlab/gonomics/vcf"
+	"log"
+)
+
+func divergenceSpectrum(inBed string, inVcf string, outFile string) {
+	var overlappingIntervals []interval.Interval
+	var currSpectrum []int
+	var err error
+
+	variants, _ := vcf.Read(inVcf)
+	varIntervals := make([]interval.Interval, 0)
+	for i := range variants {
+		varIntervals = append(varIntervals, variants[i])
+	}
+
+	tree := interval.BuildTree(varIntervals)
+	bedChan := bed.GoReadToChan(inBed)
+	out := fileio.EasyCreate(outFile)
+
+	for i := range bedChan {
+		overlappingIntervals = interval.Query(tree, i, "any")
+		currSpectrum = make([]int, 6)
+		for j := range overlappingIntervals {
+			currSpectrum[mutationType(overlappingIntervals[j].(vcf.Vcf))]++
+		}
+		i.Annotation = make([]string, 6) //replace whatever's in the input bed annotation with our output
+		for k := range currSpectrum {
+			i.Annotation[k] = fmt.Sprint(currSpectrum[k])
+		}
+		i.FieldsInitialized = 13
+		bed.WriteBed(out, i)
+	}
+	err = out.Close()
+	exception.PanicOnErr(err)
+}
+
+// mutationType returns an int corresponding to six categories of mutations. (0: A->G/T->C) (1: G->A/C->T) (2: A->T/T->A) (3: G->C/C->G) (4: A->C/T->G) (5: C->A/G->T)
```

Comment:
Would it be helpful to include info on what the numbers 0-5 mean in the usage message?

Reply:
I agree, the usage statement should have this list in it